packages/core/lib/combo/decompress.ts

```diff
@@ -57,7 +57,13 @@ const checkGameHash = (game: Game, rom: Buffer) => {
   const h = CRC32.buf(rom, 0) >>> 0;
   const hashes = CONFIG[game].crc32;
   if (!hashes.includes(h)) {
-    throw new Error(`Bad hash for ${game}, got ${h.toString(16)}`);
+    let romInfo = '';
+    if (game == 'oot') {
+      romInfo = '. For OOT, use a ROM with version 1.0, U or J.';
+    } else if (game == 'mm') {
+      romInfo = '. For MM, use a ROM with version U.';
+    } 
+    throw new Error(`Incompatable ROM file for ${game} (hash: ${h.toString(16)})${romInfo}`);
```

Comment:
```suggestion
    let romInfo = '';
    if (game == 'oot') {
      romInfo = 'For OOT, use a ROM with version 1.0, U or J.';
    } else if (game == 'mm') {
      romInfo = 'For MM, use a ROM with version U.';
    } 
    throw new Error(`Incompitable ROM file for ${game} (hash: ${h.toString(16)}). ${romInfo}`);
```

Reply:
Your typo correction is just a different typo. I've made a new commit that corrects the correction, as well as incorporating the movement of punctuation.

I'm not sure about the trailing space that would occur if the game is neither oot or mm... but it's as much of a non-issue as it is for a non-issue to be. It is the most non-issuey non-issue.